c/blake3_llfio.cpp

```diff
@@ -0,0 +1,103 @@
+#include <variant>
+
+#include <llfio.hpp>
+
+#include "blake3.h"
+#include "blake3_impl.h"
+
+namespace llfio = LLFIO_V2_NAMESPACE;
+
+template <class... Ts> struct overloads : Ts... {
+  using Ts::operator()...;
+};
+// NOTE: deduction guide only needed for Apple Clang now.
+template <class... Ts> overloads(Ts...) -> overloads<Ts...>;
+
+INLINE auto copy_wide(blake3_hasher *self, llfio::path_view path,
+                      llfio::file_handle &file) noexcept
+    -> llfio::result<std::uint64_t> {
+  std::array<std::byte, 65535> buffer{};
```

Comment:
```suggestion
  std::array<std::byte, 65535> buffer;
```
I think this is a case which warrants uninitialized memory.

`#include <array>` is missing, alternatively switch to a plain old C-array.

Reply:
Noted. I will change this.